### PR TITLE
Resolve b10t217

### PR DESCRIPTION
**Pull request by wykthor-btracker**

### CODE REVIEW

File: src/Componentes/ListCompaniesControlled/index.js [hunk 1]
```diff
@@ -30,28 +31,53 @@ export default class ListCompaniesControlled extends Component {
...
+            return (
+               <MenuItem
+                  key={company.cpn_cod}
+                  value={company.cpn_cod}
+               >
+                  {company.cpn_name}
+               </MenuItem>);
+         })}
+      </ValidationTextField>
+      // return <Form.Group controlId={this.props.controlId} /*"companyId"*/>
```
Comment (MarcosFabriciio) [inline src/Componentes/ListCompaniesControlled/index.js:59]: Remove commented code

File: src/Componentes/ListJourneysControlled/index.js [hunk 2]
```diff
@@ -27,43 +29,68 @@ export default class ListJourneysControlled extends Component {
...
+            return (
+               <MenuItem
+                  key={journey.jny_cod}
+                  value={journey.jny_cod}
+               >
+                  {journey.jny_name}
+               </MenuItem>);
+         })}
+      </ValidationTextField>
+      // return <Form.Group controlId={this.props.controlId} /*"companyId"*/>
```
Comment (MarcosFabriciio) [inline src/Componentes/ListJourneysControlled/index.js:71]: Remove commented code

File: src/Componentes/ListTypeDemand/index.js [hunk 3]
```diff
@@ -37,44 +39,72 @@ export default class ListTypeDemand extends Component {
...
+            return (
+               <MenuItem
+                  key={typeDemand.tdm_cod}
+                  value={typeDemand.tdm_cod}
+               >
+                  {typeDemand.tdm_name}
+               </MenuItem>);
+         })}
+      </ValidationTextField>
+      // return <Form.Group controlId={this.props.controlId} /*"companyId"*/>
```
Comment (MarcosFabriciio) [inline src/Componentes/ListTypeDemand/index.js:77]: Remove commented code

File: src/Pages/Badges/Cadastrar/Form.js [hunk 4]
```diff
@@ -3,11 +3,15 @@ import { Form, Col, Row, Button } from 'react-bootstrap'
...
 import { EditCreateForm } from '../../../Componentes/EditCreateForm/index'
-import { BooleanField, TextField } from '../../../Componentes/FormFields'
+import { BooleanField, InputTextField } from '../../../Componentes/FormFields'
 import { CheckBox } from '../../../Componentes/CheckBox'
 import ListCompaniesControlled from "../../../Componentes/ListCompaniesControlled"
 import ListJourneysControlled from "../../../Componentes/ListJourneysControlled"
 import { CircularProgress } from '@material-ui/core'
+import { BackGroundForm, BtnSalvar, TitleRegister } from '../../../styles/CommonStyles'
+import { IoChevronBackCircleSharp } from "react-icons/io5"
+// import TextField from '@mui/material/TextField'
```
Comment (MarcosFabriciio) [inline src/Pages/Badges/Cadastrar/Form.js:13]: Remove unused import

File: src/Pages/Usuarios/Cadastrar/Form.js [hunk 5]
```diff
@@ -26,99 +30,169 @@ export class UserFormBuilder extends EditCreateForm {
...
+                                 id="usr_name"
+                                 label="Nome"
+                                 type="text"
+                                 maxLength="200"
+                                 defaultValue={this.state.primaryData?.usr_name}
+                                 onChange={this.handleChange}
+                                 fullWidth
+                                 required
+                              />
+                              {/* <TextField
```
Comment (MarcosFabriciio) [inline src/Pages/Usuarios/Cadastrar/Form.js:51]: Remove commented code

**Review by MarcosFabriciio — CHANGES_REQUESTED** [review]
Check my comments on multiple files

File: src/Componentes/FormFields/index.js [hunk 6]
```diff
@@ -3,25 +3,70 @@ import { Form } from 'react-bootstrap'
...
+      required={props.required}
+      multiline={props.multiline ? true : null}
+      rows={props.rows ? props.rows : null}
+      InputLabelProps={{
+         shrink: true,
+         required: false,
+      }}
+      helperText={props.required ? "Campo Obrigatório" : null}
+   />
+   // return <Form.Group controlId={props.controlId}>
```
[the remark this answers is not in the document]
Comment (wykthor-btracker) [reply src/Componentes/FormFields/index.js:51]: Done

**Pull request merged**